src/Query/Operation/TableCreate.php

```diff
@@ -4,10 +4,11 @@
 namespace TBolier\RethinkQL\Query\Operation;
 
 use TBolier\RethinkQL\Message\MessageInterface;
+use TBolier\RethinkQL\Query\AbstractQuery;
 use TBolier\RethinkQL\RethinkInterface;
 use TBolier\RethinkQL\Types\Term\TermType;
 
-class TableCreate extends AbstractOperation
+class TableCreate extends AbstractQuery
```

Comment:
TableCreate needs to support other operations like Update and Update, etc. so it needs to extend AbstractQuery. See the rethink.local data explorer autocomplete or Java API.

Reply:
Fixed